src/booking.js

```diff
@@ -905,6 +907,7 @@ class RecrasBooking {
         </div>`;
             this.appendHtml(html);
             this.findElement('.bookPackage').addEventListener('click', this.submitBooking.bind(this));
+            this.maybeDisableBookButton();
```

Comment:
How does this line find it's place in the MR?

Reply:
This is needed when pre-filling amounts that are invalid. In that case, the event handler on the amount inputs doesn't get called and the message isn't shown (until you change an input or something else)